incubator/virtualcluster/cmd/vcctl/subcmd/create.go

```diff
@@ -218,16 +219,28 @@ func genKubeConfig(clusterNamespace, vcKbCfg string, cli client.Client, minikube
 
 // replaceServerAddr replace api server IP with the minikube gateway IP, and
 // disable TLS varification by removing the server CA
-func replaceServerAddr(kubeCfgContent []byte, cli client.Client, clusterNamespace string) ([]byte, error) {
-	minikubeIP, err := getMinikubeIP()
-	if err != nil {
-		return nil, err
-	}
-	svcNodePort, err := netutil.GetSvcNodePort(APIServerSvcName, clusterNamespace, cli)
-	if err != nil {
-		return nil, err
+func replaceServerAddr(kubeCfgContent []byte, cli client.Client, clusterNamespace, mode string) ([]byte, error) {
+	var newStr string
```

Comment:
what if mode == "" will vc-manager do

Reply:
we decide to replace `minikube` and `ack` mode by more general `nodeport` and `loadbalancer` mode.